### DISCUSSION TITLE
Function to stop prioritising bad placed Nodes in Router Role

### DISCUSSION THREAD

**Discussion by SimbimChimbetov** [meshtastic/firmware#4854] [Ideas]
Is it possible to have an option to ignore the prioritisation of selected routers and treat it like a client?
For vary bad placed routers, generating unnecessary hops where the operator doesn't listen and/or doesn't care or knows better.

Or a function to set priorities manually for known good placed Routers.

**Comment by shalberd** [comment]
@SimbimChimbetov you can add the routers that are hindering you to your node's Ignore List, that should help.
